package.json

```diff
@@ -36,6 +42,8 @@
     "react": "^0.14.3",
     "react-addons-test-utils": "^0.14.8",
     "react-dom": "^0.14.3",
+    "react-with-styles": "^1.0.0",
+    "react-with-styles-interface-aphrodite": "^1.0.0",
```

Comment:
From what I can tell, you are actually using these as dependencies, not devDependencies. As the code is currently written, these both would need to be moved to dependencies.

This is not ideal, however, because it couples this package to Aphrodite, which we want to avoid.